### PR TITLE
Fixed some documentation on the GET homework

### PR DESCRIPTION
**Pull request by MJTSgamer**
Also documeted 2 undocumented endpoints

### CODE REVIEW

File: homework/README.md [hunk 1]
```diff
@@ -765,7 +765,7 @@ Receives the homework from weeks after a specified date.
 
 ## Updating information
 
-### 1. Homework Made `PUT /rest/v1/swigemaakt/id`
+### 1. Homework Made `PUT /rest/v1/swigemaakt/[id]`
```
Comment (Underlyingglitch) [inline homework/README.md:768]: Good addition

File: README.md [hunk 2]
```diff
@@ -961,14 +942,145 @@ Depending on the additional parameters, some of the items in the result may not
...
 ### Undocumented:
 
 - `GET /rest/v1/medewerkers/ontvangers`
-- `GET/rest/v1/account/2555374351`
 - `GET/rest/v1/maatregeltoekenningen`
 - `GET/rest/v1/leerlingadresseringen`
-- `GET/rest/v1/schooljaren/huidig`
 - `GET/rest/v1/vakkeuzes`
-- `GET/rest/v1/verzorgers/`
+- `GET/rest/v1/verzorgers/` (probably 403 Forbidden)
```
Comment (Underlyingglitch) [inline README.md:1084]: Has this been confirmed yet? Is it maybe account dependant?

**Review by Underlyingglitch — APPROVED** [review]
Added documentation looks good. Removing the Parameters, Returns and Example links from the top of the page will improve readability. 

Take a look at my other comments though.

File: README.md [hunk 3]
```diff
@@ -961,14 +942,145 @@ Depending on the additional parameters, some of the items in the result may not
...
+#### Returns
+
+```json
+{
+  "items": [
+    {
+      "$type": "onderwijsinrichting.RSchooljaar",
+      "links": [
+        {
+          "id": 40851958, //this id is for everyone the same (in this case for year 2022/2023)
```
Comment (BraveChicken1) [inline README.md:1047]: Is this ID the same for everyone or only at your school?

File: README.md [hunk 4]
```diff
@@ -961,14 +942,145 @@ Depending on the additional parameters, some of the items in the result may not
...
+    }
+  ]
+}
+```
+
+---
+
+### Account: `GET /rest/v1/schooljaren/[huidig]`
+
+fetches information about the current school year
```
Comment (BraveChicken1) [inline README.md:1027]:
```suggestion
Fetches information about a school year
```

File: README.md (hunk 3 above)
Comment (MJTSgamer) [reply README.md:1047]: I'm pretty sure, but for certainty you can just remove it.

File: README.md (hunk 2 above)
Comment (MJTSgamer) [reply README.md:1084]: I tried with 2 accounts. But it is possible that you can only access it with an adim account, which I don't have access too.

File: README.md (hunk 3 above)
Comment (BraveChicken1) [reply README.md:1047]: I just checked, it's the same for me, so it's probably the same for everyone

File: README.md [hunk 5]
```diff
@@ -961,14 +942,145 @@ Depending on the additional parameters, some of the items in the result may not
...
+
+#### Parameters
+
+| Name          | Type   | Value                 |
+|---------------|--------|-----------------------|
+| huidig        | URL    | huidig                |
```
[the remark this answers is not in the document]
Comment (BraveChicken1) [reply README.md:1033]: Yeah that's right!

File: README.md [hunk 6]
```diff
@@ -16,35 +16,16 @@
...
-      - [Returns](#returns-5)
-      - [Example](#example-4)
     - [Absence Reports: `GET /rest/v1/absentiemeldingen`](#absence-reports-get-restv1absentiemeldingen)
-      - [Parameters](#parameters-6)
-      - [Returns](#returns-6)
     - [Study Guides: `GET /rest/v1/studiewijzers`](#study-guides-get-restv1studiewijzers)
-      - [Parameters](#parameters-7)
-      - [Returns](#returns-7)
+    - [Study Guides: `GET /rest/v1/account`](#account-get-restv1account--get-restv1accountid)
+    - [Study Guides: `GET /rest/v1/schooljaren`](#account-get-restv1schooljarenhuidig)
```
Comment (BraveChicken1) [inline README.md:28]: This link no longer works

Comment (MJTSgamer) [reply README.md:28]: Wait really, like since today?

Comment (BraveChicken1) [reply README.md:28]: The header has been changed, but the link hasn't

Comment (MJTSgamer) [reply README.md:28]: well i tested the api endpoint and it seems to work for me, it doesn't have any headers
Schermafbeelding 2022-10-18 194430

Comment (BraveChicken1) [reply README.md:28]: I meant the header in the documentation :)

Comment (MJTSgamer) [reply README.md]: Ohhh, my bad. But I don't know how to change it in this pull request... first time.

File: README.md [hunk 7]
```diff
@@ -24,8 +24,8 @@
     - [Schedule: `GET /rest/v1/afspraken`](#schedule-get-restv1afspraken)
     - [Absence Reports: `GET /rest/v1/absentiemeldingen`](#absence-reports-get-restv1absentiemeldingen)
     - [Study Guides: `GET /rest/v1/studiewijzers`](#study-guides-get-restv1studiewijzers)
-    - [Study Guides: `GET /rest/v1/account`](#account-get-restv1account--get-restv1accountid)
-    - [Study Guides: `GET /rest/v1/schooljaren`](#account-get-restv1schooljarenhuidig)
+    - [user account: `GET /rest/v1/account`](#account-get-restv1account--get-restv1accountid)
```
Comment (BraveChicken1) [inline README.md:27]:
```suggestion
    - [User Account: `GET /rest/v1/account`](#account-get-restv1account--get-restv1accountid)
```

File: README.md [hunk 8]
```diff
@@ -24,8 +24,8 @@
...
+    - [school years: `GET /rest/v1/schooljaren`](#schooljaren-get-restv1schooljaren--get-restv1schooljarenid)
```
Comment (BraveChicken1) [inline README.md:28]:
```suggestion
    - [School Years: `GET /rest/v1/schooljaren`](#schooljaren-get-restv1schooljaren--get-restv1schooljarenid)
```

**Review by BraveChicken1 — CHANGES_REQUESTED** [review]

**Comment by BraveChicken1** [comment]
Please resolve the merge conflicts

File: README.md [hunk 9]
```diff
@@ -16,44 +16,17 @@
...
-      - [Parameters](#parameters-6)
-      - [Returns](#returns-6)
-      - [Example](#example-5)
     - [Absence Reports: `GET /rest/v1/absentiemeldingen`](#absence-reports-get-restv1absentiemeldingen)
-      - [Parameters](#parameters-7)
-      - [Returns](#returns-7)
     - [Study Guides: `GET /rest/v1/studiewijzers`](#study-guides-get-restv1studiewijzers)
-      - [Parameters](#parameters-8)
-      - [Returns](#returns-8)
-    - [Subjects: `GET /rest/v1/vakken`](#subjects-get-restv1vakken)
```
Comment (BraveChicken1) [inline README.md:54]: Please don't remove Subjects

Comment (MJTSgamer) [reply README.md:54]: oops, sorry

File: README.md [hunk 10]
```diff
@@ -1134,14 +1108,146 @@ Fetches the subjects for the user
...
+
+#### Parameters
+
+| Name          | Type   | Value                 |
+|---------------|--------|-----------------------|
+| id            | URL    | [id]                  |
+| id            | URL    | huidig                |
+| Authorization | Header | Bearer [access_token] |
+
+when you want info about specificly the current school year add /huidig to the url
```
Comment (BraveChicken1) [inline README.md:1203]:
```suggestion
When you want info about the current school year add /huidig to the url
```

File: README.md [hunk 11]
```diff
@@ -1134,14 +1108,146 @@ Fetches the subjects for the user
...
 ### Undocumented:
 
 - `GET /rest/v1/medewerkers/ontvangers`
-- `GET/rest/v1/account/2555374351`
 - `GET/rest/v1/maatregeltoekenningen`
 - `GET/rest/v1/leerlingadresseringen`
-- `GET/rest/v1/schooljaren/huidig`
 - `GET/rest/v1/vakkeuzes`
-- `GET/rest/v1/verzorgers/`
+- `GET/rest/v1/verzorgers/` (probably 403 Forbidden)
```
Comment (BraveChicken1) [inline README.md:1251]: We should probably remove the `probably 403 Forbidden` as this is a valid endpoint, but just not accessible with a student account

**Review by BraveChicken1 — CHANGES_REQUESTED** [review]

File: README.md (hunk 11 above)
Comment (MJTSgamer) [reply README.md]: done ;)

**Comment by elisaado** [comment]
huts

**Pull request merged**